pkg/configtx/orderer.go

```diff
@@ -17,11 +17,9 @@ import (
 	cb "github.com/hyperledger/fabric-protos-go/common"
 	ob "github.com/hyperledger/fabric-protos-go/orderer"
 	eb "github.com/hyperledger/fabric-protos-go/orderer/etcdraft"
+	"github.com/hyperledger/fabric/pkg/configtx/orderer"
 )
 
-// ConsensusState defines the orderer mode of operation.
-type ConsensusState string
-
 // Orderer configures the ordering service behavior for a channel.
 type Orderer struct {
```

Comment:
Matt suggested it would look better to namespace orderer subtypes in a separate package and I do agree it looks better now. The orderer type itself hasn't moved since it feels appropriate to namespace it as `configtx.Orderer` instead of `orderer.Orderer`. We're keeping a close look though on the upcoming work with chaining APIs and it may make sense to move more into the orderer subpackage as we progress through that but I think this is a good starting point for now